crates/matrix-sdk-base/Cargo.toml

```diff
@@ -53,13 +54,18 @@ ruma = { version = "0.6.1", features = ["client-api-c", "js", "signatures"] }
 ruma = { version = "0.6.1", features = ["client-api-c", "signatures"] }
 
 [dev-dependencies]
-futures = { version = "0.3.21", default-features = false, features = ["executor"] }
+futures = { version = "0.3.21", default-features = false, features = [
+  "executor",
+] }
 tracing = { version = "0.1.26", features = ["log"] }
 http = "0.2.6"
 assign = "1.1.1"
 env_logger = "0.9.0"
 matrix-sdk-test = { version = "0.4.0", path = "../matrix-sdk-test" }
-tokio = { version = "1.17.0", default-features = false, features = ["rt-multi-thread", "macros"] }
+tokio = { version = "1.17.0", default-features = false, features = [
```

Comment:
Same as the previous one.

Reply:
My bad, I haven’t realized the autoformatter I have set up for toml formatted this file too